FlightPlanner.Web/FlightPlanner.Web/Storage/FlightStorage.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Linq;
+using FlightPlanner.Web.DbContext;
+using FlightPlanner.Web.Interface;
+using FlightPlanner.Web.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace FlightPlanner.Web.Storage
+{
+    public static class FlightStorage
+    {
+        private static readonly AirportStorage _airportStorage = new();
+
+        public static Flight GetFlight(int id, FlightPlannerDbContext context)
+        {
+            return FindFlight(id, context);
+        }
+
+        public static Flight FindFlight(int id, FlightPlannerDbContext context)
+        {
+            return context.Flights
+                .Include(a => a.To)
+                .Include(a => a.From)
+                .SingleOrDefault(f => f.Id == id);
+        }
+
+        public static Flight CreateFlight(AddFlightRequest flightRequest)
+        {
+            
+            Flight flight = new Flight
+            {
+                ArrivalTime = flightRequest.ArrivalTime,
+                Carrier = flightRequest.Carrier,
+                DepartureTime = flightRequest.DepartureTime,
+                From = flightRequest.From,
+                To = flightRequest.To
+            };
+
+            return flight;
+        }
+
+        public static bool IsSameFlight(AddFlightRequest flightRequest, FlightPlannerDbContext context)
+        {
+            Flight result = context.Flights
+                .Include(a => a.To)
+                .Include(a => a.From)
+                .FirstOrDefault(f => f.ArrivalTime == flightRequest.ArrivalTime 
+                                     && f.Carrier == flightRequest.Carrier
+                                     && f.DepartureTime == flightRequest.DepartureTime
+                                     && f.From.AirportCode == flightRequest.From.AirportCode
+                                     && f.From.City == flightRequest.From.City
+                                     && f.From.Country == flightRequest.From.Country
+                                     && f.To.AirportCode == flightRequest.To.AirportCode
+                                     && f.To.City == flightRequest.To.City
+                                     && f.To.Country == flightRequest.To.Country);
+
+            return result != null;
+        }
+
+        public static bool IsValidFlight(AddFlightRequest flightRequest)
+        {
+            //check for null and empty values
+            bool result = String.IsNullOrEmpty(flightRequest.ArrivalTime)
+              || String.IsNullOrEmpty(flightRequest.DepartureTime)
+              || String.IsNullOrEmpty(flightRequest.Carrier)
+              || flightRequest.From == null
+              || flightRequest.To == null;
+
+            if (result)
```

Comment:
if (result) -- šis neatbild ne uz kādu jautājumu

Reply:
Saliku mainīgos naming ar nozīmi